botResponses/botResponses.js

```diff
@@ -156,6 +156,22 @@ function botResponseDontGiveUp({
   )
 }
 
+function botResponseJustDoIt({
```

Comment:
gotta export this function at the bottom of the file

(I should really clean that up :/)

Reply:
Should be ok now :eyes: